client/dashboard/app/router.tsx

```diff
@@ -270,14 +270,30 @@ const siteLogsChildRoutes = [ siteLogsIndexRoute, siteLogsPhpRoute, siteLogsServ
 const siteBackupsRoute = createRoute( {
 	getParentRoute: () => siteRoute,
 	path: 'backups',
+} );
```

Comment:
I think we should continue rendering `sites/backups` here so the upsell is shown under all backups-related routes.
Next, we should pass the `<Outlet />` component to the main prop of `<CalloutOverlay />`.

With this setup:
* `siteBackupsIndexRoute` will render the data views.
* `siteBackupRestoreRoute` will render the restore component.

Both routes can still remain gated under the upsell.